src/parser/entities/ray/types.ts

```diff
@@ -0,0 +1,11 @@
+import type { Point3D } from '../../../types';
+import type { CommonDxfEntity } from '../shared';
+
+export interface RayEntity extends CommonDxfEntity {
+    type: 'RAY';
+    subclassMarker: 'AcDbRay';
+    /** Start point (in WCS) */
+    position: Point3D;
+    /** Unit direction vector (in WCS) */
+    direction: Point3D;
```

Comment:
Properties are slightly different to #25. I prefer more concise and simple name. Also similar meanings should be similar forms, and there is a `direction` property in MTEXT. Same for `position` which can be found easily in this repository.